vite.config.ts

```diff
@@ -1,11 +1,22 @@
+import path from "path";
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import copy from "rollup-plugin-copy";
 
+const PORT = process.env.VITE_DEV_SERVER_PORT as undefined|number;
```

Comment:
This typing is incorrect, `PORT` is either a `string` or `undefined` as all `process.env.*` are strings.

Reply:
Could you help, please - how it must be? because Vite expects a number.
<img width="1020" alt="Screenshot 2024-07-26 at 15 43 55" src="https://github.com/user-attachments/assets/2f626609-a065-4939-955d-4b5d210efea8">

Reply:
@zpawn you'll need to cast the env var as a number

Reply:
Fixed